402.md

```diff
@@ -0,0 +1,71 @@
+# NIP-402
+
+## Classifieds
+
+`draft` `optional` `author:erskingardner`
+
+This NIP defines `kind:30402`: a parameterized replaceable event to describe classified listings that list any arbitrary product, service, or other thing for sale or offer and includes enough structured metadata to make them useful.
+
+The category of classifieds includes a very broad range of physical goods, services, work opportunities, rentals, free giveaways, personals, etc. and is distinct from the more strictly structured marketplaces defined in [NIP-15](https://github.com/nostr-protocol/nips/blob/master/15.md) that often sell many units of specific products through very specific channels.
+
+The structure of these events is very similar to [NIP-23](https://github.com/nostr-protocol/nips/blob/master/23.md) long-form content events.
+
+### Draft / Inactive Listings
+
+`kind:30403` has the same structure as `kind:30402` and is used to save draft or inactive classified listings.
+
+### Content
+
+The `.content` field should be a description of what is being offered and by whom. These events should be a string in Markdown syntax.
+
+### Author
+
+The `.pubkey` field of these events are treated as the party creating the listing.
+
+### Metadata
+
+- For "tags"/"hashtags" (i.e. categories or keywords of relevance for the listing) the `"t"` event tag should be used, as per [NIP-12](https://github.com/nostr-protocol/nips/blob/master/12.md).
+
+The following tags, used for structured metadata, are standardized and SHOULD be included. Other tags may be added as necessary.
+
+- `"title"`, a title for the listing
+- `"summary"`, for short tagline or summary for the listing
+- `"published_at"`, for the timestamp (in unix seconds – converted to string) of the first time the listing was published.
```

Comment:
Redundancy with NIP01’s `created_at` field or usage is expected to be different ?

Reply:
It's not redundancy. NIP-23 long-form articles do the same thing. `published_at` is the date of first publish. Created at is the creation time of the event itself. This is a way of tracking the initial publish time through edits.

Reply:
>  published_at is the date of first publish. 

If the rationale is it to be used for each event NIP33 replacement it can be added. I can see how it can be used a “beginning” temporal point by the relay to prune one of the replaced instance of the event after a while (e.g “delete my  good announcement after X, whatever the number of replacement”). Rationale could be added.